### DISCUSSION TITLE
连接go-cqhttp时websocket连接失败，提示

### DISCUSSION THREAD

**Discussion by ZephyrYoung-eYuan** [Tsuk1ko/CQ-picfinder-robot#263] [问题求助] [answered]
go-cqhttp一直在提示
failed to accept WebSocket connection: request Origin "[object Object]" is not authorized for Host "127.0.0.1:6700"
![image](https://user-images.githubusercontent.com/64537579/141666094-6fda6e63-ba92-4134-8bf0-afd9d0ea0d52.png)

bot这边也提示
![image](https://user-images.githubusercontent.com/64537579/141666087-0d88a8fa-e16a-4350-aca3-3113109745d5.png)

cq配置方面也检查过应该没有问题
![image](https://user-images.githubusercontent.com/64537579/141666121-e28946e6-b9b8-4bf4-84ee-4379a8e49897.png)

并没有修改过bot的配置
![image](https://user-images.githubusercontent.com/64537579/141666111-6606f45b-8a1f-4327-b40f-2419b3213ff2.png)

换电脑也不好使，求助dalao们，提前感谢！

**Comment by ZephyrYoung-eYuan** [comment]
cq的token配置
![image](https://user-images.githubusercontent.com/64537579/141666275-507384e0-7188-421d-8aa3-5c6f4a43bfef.png)

**Comment by Miuzarte** [comment, marked as answer]
不要用v1.0.0-beta8
![image](https://user-images.githubusercontent.com/66856838/141668590-27178a54-2267-40af-9814-428e3890ef17.png)

**Reply by ZephyrYoung-eYuan** [reply]
谢谢，已经解决了，cq的beta8版本的问题，在go-cqhttp的最新action里修正了这个bug

**Comment by Tsuk1ko** [comment]
嗯，先用 v1.0.0-beta7
